packages/admin/admin/src/mui/menu/Item.styles.ts

```diff
@@ -120,6 +119,8 @@ export const Root = createComponentSlot(ListItemButton)<MenuItemClassKey, OwnerS
             .${listItemTextClasses.primary} {
                 font-size: 16px;
                 line-height: 20px;
+                margin-left: 12px;
+                font-weight: 450;
```

Comment:
Shouldn't this come from the theme?

Reply:
We don't have fontWeights in the theme anymore, so it can't. Also, UX didn't use a default Typography here but custom values.